deep_qa/common/util.py

```diff
@@ -30,3 +30,42 @@ def add_noise_to_dict_values(dictionary: Dict[Any, float], noise_param: float) -
         noise = random.uniform(-noise_value, noise_value)
         new_dict[key] = value + noise
     return new_dict
+
+
+def clean_layer_name(input_name: str,
+                     keep_right_of_last_backslash: bool=None,
```

Comment:
If it's None, we don't split at all though - if it's true, we take the right hand side after splitting, if false we take the left hand side, but I agree that it's confusing.